.azure/pipelines/ci.yml

```diff
@@ -597,9 +608,20 @@ stages:
       agentOs: Windows
       timeoutInMinutes: 180
       steps:
-      - script: .\restore.cmd -ci
-        displayName: Restore
-      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsHelixJob=true /p:IsHelixDaily=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
+      # Build the x86 shared framework
+      # TODO: make it possible to build for one Windows architecture at a time
+      # This is going to actually build x86 native assets. See https://github.com/aspnet/AspNetCore/issues/7196
+      - script: ./build.cmd
+                -ci
+                -arch x86
```

Comment:
Shouldn't this be 'x64' for 64-bit testing?

Reply:
I only saw an x86 shared framework in the ci.yml, so I assume it doesn't matter?

Reply:
The Windows x64 / x86 build step's comments are a bit confusing. The first `./build.cmd` invocation creates the x64 shared runtime. The second (commented) one adds just the x86-specific bits.

I'm surprised the tests succeeded with the x86 shared runtime. Does anything actually test using it?

Reply:
Yes this is on by default so all the shared framework bits are getting copied onto the helix machines for every test, so it doesn't seem to be causing any issues.

Reply:
My point is the created x86 shared framework here may not be used at all because it's not the bitness the tests need. Not causing issues isn't the same thing as enabling the testing we need.

Reply:
Sure what's the fix just specify arch x64?

Reply:
I suggest adding a test that won't run unless it executes against the expected shared framework. Maybe something that looks at the location of a loaded SharedFX assembly. That'll help show what's working (or not) here.

Put another way, I'd rather not see this new infrastructure need further changes as soon as we try to use it.

Reply:
This is prerequisite work for template testing, I don't think we need to add a new test for that, the only reason we need this is because its a requirement for templates which will fail if they aren't running against latest shared framework bits.

Reply:
Rewording what I said: This doesn't look like it should work. Need to show that it is working w/ Win x64 test.

But, sure, one way to do that would be to enable template tests in the same PR.